lib/code_fund_web/controllers/property_controller.ex

```diff
@@ -26,15 +26,18 @@ defmodule CodeFundWeb.PropertyController do
 
   def new(conn, _params) do
     current_user = conn.assigns.current_user
-    form = create_form(PropertyType, %Property{}, %{}, user: current_user)
+
+    form =
+      create_form(PropertyType, %Property{}, %{}, user: current_user, current_user: current_user)
+
     render(conn, "new.html", form: form)
   end
 
   def create(conn, %{"property" => property_params}) do
     current_user = conn.assigns.current_user
 
     PropertyType
-    |> create_form(%Property{}, property_params, user: current_user)
+    |> create_form(%Property{}, property_params, user: current_user, current_user: current_user)
```

Comment:
is this redundant when we are already passing in current_user as user?

Reply:
so, unfortunately, i'm not sure if it's safe to use `user` here since there's one `formex.form` for all the actions and i don't want to overwrite the owner of properties on an update so I just pass/call current_user in each case.